runelite-client/src/main/java/net/runelite/client/plugins/worldmap/FarmingPatchLocation.java

```diff
@@ -40,9 +40,9 @@
 		new WorldPoint(3598, 3524, 0),
 		new WorldPoint(3052, 3309, 0),
 		new WorldPoint(2810, 3462, 0),
-		new WorldPoint(2663, 3375, 0),
-		new WorldPoint(3289, 6100, 0)
+		new WorldPoint(2663, 3375, 0)
 	),
+	ALLOTMENT_FLOWER("Allotment/Flower", new WorldPoint(3289, 6100, 0)),
```

Comment:
This should be above `ALLOTMENT_HERB_FLOWER` as the entries are ordered alphabetically.

Reply:
Roger that. Also thanks for the tip about space!